xyz.safewords.midiconn.yml

```diff
@@ -0,0 +1,63 @@
+command: midiconn
+id: xyz.safeworlds.midiconn
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+modules:
+  - name: SDL2
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/libsdl-org/SDL
+        tag: release-2.26.1
+  - name: fmt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DFMT_DOC=OFF
+      - -DFMT_INSTALL=ON
+      - -DFMT_TEST=OFF 
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/fmtlib/fmt
+        tag: '8.1.1'
+  - name: spdlog
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/gabime/spdlog
+        tag: v1.10.0
+  - name: rtmidi
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DRTMIDI_API_JACK=OFF
```

Comment:
how do you access MIDI then?

Reply:
RtMidi has an ALSA backend. Note, that this application is not a GUI for Jack, it does MIDI routing on its own. However, support for Jack is being considered.

Reply:
did you test it?

Reply:
Yes, the current flatpak version correctly detects the connected MIDI hardware and is able to send and receive MIDI messages.